docs/real-device-app-testing.md

```diff
@@ -60,17 +60,17 @@ In the case of **iOS**, you can install from the App Store, Test Flight or from
 
 <img loading="lazy" src={require('../assets/images/mobile-app-testing/real-devices-for-ios.webp').default} alt="Real "  className="doc_img" width="1366" height="629"/>
 
-**Step 4**: You can select your preferred region from the **Region** dropdown. Choose from the available regions i.e., North America, Europe, Asia, or select All. For assistance with using this feature, please feel free to email us at [support@lambdatest.com](mailto:support@lambdatest.com). 
+**Step 4**: You can select your preferred region from the **Region** dropdown. Choose from the available regions i.e., North America, Europe, Asia, or select All. If you would like to choose devices from specific regions, please reach out to [support@lambdatest.com](mailto:support@lambdatest.com) to get the option enabled.
 
 <img loading="lazy" src={require('../assets/images/mobile-app-testing/real-device-region-feature.webp').default} alt="Real Device Region Feature"  className="doc_img" width="1366" height="629"/>
 
-**Step 5**: Select the **Tunnel** option in the top right corner to test your locally hosted and privately hosted websites. 
+**Step 5**: Select the **Tunnel** option in the top right corner (optional) to test your locally hosted and privately hosted websites.
```

Comment:
**Step 5** itself is optional. Please add optional next to step 5 such as :

**Step 5 (Optional)**: Select the......